engine/events/mom.asm

```diff
@@ -283,7 +283,7 @@ BankOfMom:
 	ret
 
 .AskDST:
-	ld hl, UnknownText_0x16694
+	ld hl, MomVisitingText13
```

Comment:
This is not a simple sequence of texts like Oak's speech, and some already have meaningful names (MomLostGearBookletText, etc), so please make all of them that way.